From: Boudhayan Gupta <me@BaloneyGeek.com>
Date: Sun, 17 Jan 2016 15:41:07 +0000
Subject: DO NOT MERGE TO MASTER: Make save options dialog resizable
X-Git-Url: http://quickgit.kde.org/?p=spectacle.git&a=commitdiff&h=688e1e2a4642f6f43e3ce7bfd2edff0cae106dcd
---
DO NOT MERGE TO MASTER: Make save options dialog resizable

BUG: 358103
FIXED-IN: 15.12.2
---


--- a/src/Gui/KSSaveConfigDialog.cpp
+++ b/src/Gui/KSSaveConfigDialog.cpp
@@ -26,7 +26,8 @@
 
     setWindowTitle(i18n("Configure Save Options"));
     setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
-    setFixedSize(500, 600);
+    setMinimumSize(500, 600);
+    resize(500, 600);
 
     // bring up the configuration reader
 